bfd/ChangeLog.COREV

```diff
@@ -10,8 +10,12 @@
 
 2020-09-10  Pietra Ferreira  <pietra.ferreira@embecosm.com>
 
-	* bfd-in2.h: Added CORE-V hardware loop specific relocations.
-	* elfnn-riscv.c: Added relocations for CORE-V hardware loop.
-	* elfxx-riscv.c: Added relocations and amend howto lookup.
-	* bfd/riscv.c: Added BFD_RELOC_RISCV_CVPCREL_UI12 and
-	BFD_RELOC_RISCV_CVPCREL_URS1.
+	* bfd-in2.h: Add hardware loop specific relocations -
```

Comment:
This line should just say 'Regenerated.' Since this is a file which is only edited via `make headers`, your change here is just regenerating it, and convention here is to write 'Regenerated.' for such a change.

Reply:
@simonpcook Thanks for reviewing! I have made this change.

Reply:
It looks like you still have extra text, I mean that the line should just say `Regenerated.` and nothing else. What changes caused the regeneration of other areas is clear by the other changes in that commit.

Reply:
@simonpcook Ah I see, sorry I misunderstood. Updated now.